controllers/articles.js

```diff
@@ -0,0 +1,195 @@
+import crypto from 'crypto';
+import models from '../models/index';
+
+const tempUser = '2dd9e22b-d19d-4501-a6b5-00bb4ebd9b3e'; // This line has to be deleted.
```

Comment:
We have JWT authentication working you should be getting this id from the user object added to the request after the token is verified

Reply:
Since we are collaborating on this branch with @niyobobo, I will work with him to solve this issue. First will get to know why he was using the tempUser here.